telemetry.md

```diff
@@ -24,6 +24,7 @@ When the telemetry collection feature is enabled in TiDB, the TiDB cluster colle
 - Deployment characteristics, such as the size of hardware (CPU, memory, disk), TiDB components versions, OS name.
 - The status of query requests in the system, such as the number of query requests and the duration.
 - Component usage, for example, whether the Async Commit feature is in use or not.
+- Builtin functions usage information, including the count of functions used in SQL queries.
```

Comment:
```suggestion
- Usage information of builtin functions, including the average number of functions used per SQL query.
```

```suggestion
- Usage information of builtin functions, including the count of functions used in SQL queries.
```

Reply:
Actually it is a count per function. 

I'm not sure how to explain it clearly. What do you think about this?

Reply:
Would this be ok?
```suggestion
- Usage information of builtin functions, including per-function statistics on how often the functon gets called. The arguments to the functions are not included.
```

Reply:
Nice, I'll modify the part in Chinese version according to this as well.